### PR TITLE
Move steps to be under default extension

### PR DESCRIPTION
**Pull request by twsouthwick**
This change moves all the steps to be under the default extension. This is how things are currently packaged/architected, but the organization on disk and in the solution file did not make that easy to understand.

### CODE REVIEW

**Comment by twsouthwick** [comment]
Sorry for the large change, but it is mostly just moves and no actual code changes

File: Microsoft.DotNet.UpgradeAssistant.sln [hunk 1]
```diff
@@ -168,11 +164,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.UpgradeAss
...
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "maui", "maui", "{7304F25F-0F4D-450F-A529-3C7AFE9D98CA}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.DotNet.UpgradeAssistant.Extensions.Maui.Tests", "tests\extensions\maui\Microsoft.DotNet.UpgradeAssistant.Extensions.Maui.Tests\Microsoft.DotNet.UpgradeAssistant.Extensions.Maui.Tests.csproj", "{CEDAAB8D-613C-4A3F-8114-7D6B480F1EE3}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.UpgradeAssistant.Extensions.Maui.Tests", "tests\extensions\maui\Microsoft.DotNet.UpgradeAssistant.Extensions.Maui.Tests\Microsoft.DotNet.UpgradeAssistant.Extensions.Maui.Tests.csproj", "{CEDAAB8D-613C-4A3F-8114-7D6B480F1EE3}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.DotNet.UpgradeAssistant.Steps.Source.Tests", "tests\steps\Microsoft.DotNet.UpgradeAssistant.Steps.Source.Tests\Microsoft.DotNet.UpgradeAssistant.Steps.Source.Tests.csproj", "{60A96474-2199-49B5-AAB5-FA7AF2C0B595}"
```
Comment (sunandabalu) [inline Microsoft.DotNet.UpgradeAssistant.sln:173]: This should move under extensions too.

Comment (twsouthwick) [reply Microsoft.DotNet.UpgradeAssistant.sln:173]: Maui is under extensions. Are you referring to something else?

Comment (twsouthwick) [reply Microsoft.DotNet.UpgradeAssistant.sln]: Oh I see what you meant! Fixed

**Review by sunandabalu — APPROVED** [review]

**Pull request merged**